.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
I don't think we should add IDE specific configuration here. This is just a slippery slope. Allow VSCode, then Sublime, then Atom, then PhpStorm...

Reply:
Don't know why this file upload here! Please ignore.